### PR TITLE
[Feature] Historic rates table body

### PR DESCRIPTION
**Pull request by ghost**
#### What does this PR do?
- Renders the body of the `HistoricRatesTable`

#### Description of Task to be completed?
- Create module for manipulating data.
- Fix ApolloProvider prop-type failure
- Fetch Locations in HistoricRates table
- Format date before rendering
- Give data render aesthetic value
- Render data on HistoricRatesScreen

#### Screenshot
screen shot 2018-06-09 at 12 36 16 pm

### CODE REVIEW

File: src/components/HistoricRatesBody.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import React, { PureComponent } from 'react';
+import { View, FlatList, ActivityIndicator } from 'react-native';
+import { HistoricRatesStyles as styles } from '../styles';
+import { HistoricRatesItem } from '.';
+import { Query } from 'react-apollo';
+import { GET_COMPUTED_RATES } from '../operations';
+import { sortRatesByDate } from '../dataApi';
+import Whoops from './Whoops';
```
Comment (omorilewa) [inline src/components/HistoricRatesBody.js:8]: Really?

File: src/components/HistoricRatesBody.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+export default class HistoricRatesBody extends PureComponent {
+
+  render() {
+    return (
+      <Query query={GET_COMPUTED_RATES}>
+        {({ data, loading, error }) => {
+          if (error) {
+            return <Whoops message="Error while fetching rates" />;
+          } else if (loading) {
```
Comment (omorilewa) [inline src/components/HistoricRatesBody.js:18]: Why an else-if ?

**Comment by omorilewa** [comment]
Coverage decreased after the tests?

File: .babelrc [hunk 3]
```diff
@@ -1,3 +1,10 @@
 {
-  "presets": ["react-native"]
+  "presets": ["react-native"],
+  "plugins": [
+    ["module-resolver", {
```
Comment (omorilewa) [inline .babelrc:4]: I am interested in knowing why we need a module resolver.

Comment (ghost) [reply .babelrc:4]: The tests fail, producing an `Cannot find module 'react-dom/server' from 'react-apollo.umd.js'` message on the console and on travis. When I searched for solutions, I found that as a fix. I am open to a sleeker alternative.

File: src/components/HistoricRatesBody.js (hunk 2 above)
Comment (ghost) [reply src/components/HistoricRatesBody.js:18]: Lol. Just for the fun of it. An if works naturally, an `else if` can be used once an `if` has already been used

**Review by omorilewa — APPROVED** [review]
Good work. Approving because the files changed are many and may be difficult to track.

**Pull request merged**